BlockServer/config/xml_converter.py

```diff
@@ -419,6 +423,10 @@ def meta_from_xml(root_xml, data):
         if synoptic is not None:
             data.synoptic = synoptic.text if synoptic.text is not None else ""
 
+        isProtected = root_xml.find("./" + TAG_PROTECTED)
+        if isProtected is not None:
+            data.isProtected = isProtected.text if isProtected.text is not None else ""
```

Comment:
Is the empty string valid for the `xs:boolean` type? Would be better to explicitly default to `"true"`/`"false"`

Reply:
default value set to "false".